### ISSUE TITLE
Account disabled?

### ISSUE THREAD

**Issue by xfisbest** [hitrov/oci-arm-host-capacity#66]
Hi there, i've been running this script from 3-4 days in singapore region, with 2 minutes interval, now i am getting error
`You don’t have permission to view these resources in this compartment. Try another compartment, or contact your administrator for help.`
 logs of oci.log:
`Thu 13 Apr 2023 06:02:01 PM BST { "code": "InternalError", "message": "Out of host capacity." } Thu 13 Apr 2023 06:04:01 PM BST { "code": "InternalError", "message": "Out of host capacity." } Thu 13 Apr 2023 06:06:01 PM BST { "code": "InternalError", "message": "Out of host capacity." } Thu 13 Apr 2023 06:08:01 PM BST { "code": "InternalError", "message": "Out of host capacity." } Thu 13 Apr 2023 06:10:01 PM BST { "code": "InternalError", "message": "Out of host capacity." } Thu 13 Apr 2023 06:12:01 PM BST { "code": "InternalError", "message": "Out of host capacity." } Thu 13 Apr 2023 06:14:01 PM BST { "code": "InternalError", "message": "Out of host capacity." } Thu 13 Apr 2023 06:16:01 PM BST { "code": "InternalError", "message": "Out of host capacity." } Thu 13 Apr 2023 06:18:01 PM BST { "code": "NotAuthorizedOrNotFound", "message": "Authorization failed or requested resource not found." } Thu 13 Apr 2023 06:20:01 PM BST Thu 13 Apr 2023 06:22:01 PM BST Thu 13 Apr 2023 06:24:01 PM BST Thu 13 Apr 2023 06:26:01 PM BST Thu 13 Apr 2023 06:28:01 PM BST Thu 13 Apr 2023 06:30:01 PM BST Thu 13 Apr 2023 06:32:01 PM`

Now i disabled my script. What's happening?

**Comment by hitrov** [comment]
Hey, try logging into OCI web console first

**Comment by Trngd** [comment]
Yes, I have also faced this issue. After contacting support, my account turned out to be permanently disabled and oracle refuses to provide any additional information.

**Comment by IvsonEmidio** [comment]
I'm facing this same issue, i've three acounts, just one that i've used this script has been banned, i believe OCI detects it and ban accounts after some times :/

**Comment by hitrov** [comment]
@IvsonEmidio How frequently did you configure the script to execute?

**Comment by IvsonEmidio** [comment]
> @IvsonEmidio How frequently did you configure the script to execute?

A try per minute if i'm not mistaken
